components/Styled/IconButton.js

```diff
@@ -1,11 +1,17 @@
 import styled from "styled-components";
 import ArrowLeft from "@/public/icons/ArrowSmall.svg";
+import ChevronSmall from "@/public/icons/ChevronSmall.svg";
+import Plus from "@/public/icons/Plus.svg";
+import XSmall from "@/public/icons/XSmall.svg";
 import TriangleLeft from "@/public/icons/arrow-left-3-svgrepo-com.svg";
 import TriangleRight from "@/public/icons/arrow-right-2-svgrepo-com.svg";
 import Heart from "@/public/icons/heart-svgrepo-com.svg";
 
 export default function IconButton({ onClick, style, left, right, top, fill }) {
   const buttonStyles = {
+    chevrondown: <ChevronSmall width={30} height={30} />,
+    plus: <Plus width={30} height={30} />,
+    x: <XSmall width={30} height={30} />,
     ArrowLeft: <ArrowLeft width={30} height={30} />,
     TriangleLeft: <TriangleLeft width={30} height={28} />,
     TriangleRight: <TriangleRight width={30} height={30} />,
```

Comment:
to make it more usiveral for all buttons on yourp page it would be great to implement one "Button" component and make it very versatile with the props feature of styled components :)